### PR TITLE
docs: add RELEASE.md

### PR DESCRIPTION
**Pull request by kenjis**

### CODE REVIEW

File: admin/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+> -kenjis
+
+## Changelog
+
+When generating the changelog each Pull Request to be included must have one of
+the following [labels](https://github.com/codeigniter4/shield/labels):
+- **bug** ... PRs that fix bugs
+- **enhancement** ... PRs to improve existing functionalities
+- **new feature** ... PRs for new features
+- **refactor** ... PRs to refactor
```
Comment (datamweb) [inline admin/RELEASE.md:14]:
```suggestion
- **refactor** ... PRs to refactor
- **lang** ... PRs for new/update language
```

File: admin/RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Check the resulting content. If there are items in the *Others* section which
+should be included in the changelog, add a label to the PR and regenerate
+the changelog.
+
+## Preparation
+
+* Clone **codeigniter4/shield** and resolve any necessary PRs
+```console
+git clone git@github.com:codeigniter4/shield.git
+```
```
Comment (datamweb) [inline admin/RELEASE.md:41]:
```suggestion
* Clone **codeigniter4/shield** and resolve any necessary PRs

  ```console
  git clone git@github.com:codeigniter4/shield.git
  ```
```

File: admin/RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* Fast-forward `develop` branch to catch the merge commit from `master`
+```console
+git fetch origin
+git checkout develop
+git merge origin/develop
+git merge origin/master
+git push origin HEAD # @TODO can't push to protected branch.
+```
+* Publish any Security Advisories that were resolved from private forks
+* Announce the release on the forums and Slack channel
```
Comment (datamweb) [inline admin/RELEASE.md:80]: I like what **lonnieezell**  has done before #203, I think it's auto generated.

**Review by datamweb — COMMENTED** [review]
Thanks for the PR, however I'm not sure I'm the person who should **approved**, maybe **MGatner** or **lonnieezell** would be better to see here. I have not experienced this.

File: admin/RELEASE.md (hunk 3 above)
Comment (kenjis) [reply admin/RELEASE.md:82]: Added https://github.com/codeigniter4/shield/pull/687/commits/426fecf3348dfedc314e19bff3e959219fa34b44

File: admin/RELEASE.md (hunk 2 above)
Comment (kenjis) [reply admin/RELEASE.md]: Indented.

Comment (datamweb) [reply admin/RELEASE.md]: thanks!

**Comment by kenjis** [comment]
Added notes to items that requires administrator privileges.

File: admin/RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+* Create a new branch `release-1.x.x`
+* Update **src/Auth.php** with the new version number:
+  `const SHIELD_VERSION = '1.x.x';`
+* Commit the changes with "Prep for 1.x.x release" and push to origin
+* Create a new PR from `release-1.x.x` to `develop`:
+    * Title: "Prep for 1.x.x release"
+    * Description: "Updates version references for `1.x.x`." (plus checklist)
+* Let all tests run, then review and merge the PR
+* Create a new PR from `develop` to `master`:
```
Comment (MGatner) [inline admin/RELEASE.md:59]: I wouldn't normally go through all this trouble for a package. Not opposed, but it makes the release harder.

**Review by MGatner — APPROVED** [review]

File: admin/RELEASE.md (hunk 4 above)
Comment (kenjis) [reply admin/RELEASE.md:59]: Do you mean that not creating a PR but just run `git merge`?

When releasing a package, I normally just tag in a version branch like `1.x`.

**Review by lonnieezell — APPROVED** [review]
Looks great. Thanks for pulling this together.

**Pull request merged**